src/modules/listclass.js

```diff
@@ -0,0 +1,50 @@
+export default class TodoList {
+    allTodos = [];
+
+    saveTodo() {
+      const todos = JSON.stringify(this.allTodos);
+      localStorage.setItem('todos', todos);
+    }
+
+    addTodo(description) {
+      const todo = {
+        description,
+        completed: false,
+        index: this.allTodos.length + 1,
+      };
+      this.allTodos.push(todo);
+      this.saveTodo();
+    }
+
+    deleteTodo(indx) {
+      this.allTodos.splice(indx - 1, 1);
+      this.allTodos.forEach((t) => {
+        if (t.index > indx) {
+          t.index -= 1;
+        }
+      });
+      this.saveTodo();
+    }
+
+    getStoredTodos() {
+      this.allTodos = JSON.parse(localStorage.getItem('todos'));
+    }
+
+    editTodo(newValue, index) {
+      this.allTodos[index - 1].description = newValue;
+      this.saveTodo();
+    }
+
+    completedTodo(status, index) {
```

Comment:
- [ ] Generally speaking, your code uses all the naming conventions. It would be awesome if you could use an action verb for the `completedTodo` method too.
- [ ] 
Suggestion: You can refractor it to `completeTodo` instead.

Reply:
Thank you @Sboursen, I have done the suggested changes!